integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/ad-hoc/SynchronizeTime.feature

```diff
@@ -8,14 +8,24 @@ Feature: SmartMetering AdHoc
   I want to be able to synchronize time on a device
   So time related data on the device will have reliable timestamps
 
-  Scenario: Retrieve SynchronizeTime result from a device
+  Scenario Outline: Retrieve SynchronizeTime result from a <protocol> <version> device
     Given a dlms device
-      | DeviceIdentification | TEST1024000000001 |
+      | DeviceIdentification | <deviceIdentification> |
       | DeviceType           | SMART_METER_E     |
+      | Protocol             | <protocol>             |
+      | ProtocolVersion      | <version>              |
     When receiving a get synchronize time request
-      | DeviceIdentification | TEST1024000000001 |
+      | DeviceIdentification | <deviceIdentification> |
     Then the date and time is synchronized on the device
-      | DeviceIdentification | TEST1024000000001 |
-    And the dlms device with identification "TEST1024000000001" exists with properties
+      | DeviceIdentification | <deviceIdentification> |
+    And the dlms device with identification "<deviceIdentification>" exists with properties
       | DlmsDeviceTimezone | Europe/Amsterdam |
-
```

Comment:
Its about 1,5 second but just to mention it. You could define two sets of examples and annote the larger one with @NightlyBuildOnly